tests/unit/test_rbac.py

```
@@ -0,0 +1,176 @@
+"""
+
+from __future__ import annotations
+
+import pytest
+
+from src.services.rbac_service import RBACService
+
+
+@pytest.fixture
+def rbac_service():
+    return RBACService(MagicMock())
```

Pylint (4.0.5)

[error] Undefined variable 'MagicMock'

(E0602)